server/server.go

```diff
@@ -278,7 +296,6 @@ func (u *UpdateServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Header().Set("X-Message-Signature", hex.EncodeToString(messageAuth))
 
-	w.WriteHeader(http.StatusOK)
```

Comment:
It does this by default is that right @joesis?

Reply:
Yeah that's the default.